open-craft/opencraft
# SE-2293 Switch to S3 as the default instance storage type for unit tests

Switches default storage instance type to S3 and adds Swift storage type override wherever necessary.

**JIRA tickets**: SE-2293

**Dependencies**: None

**Merge deadline**: None

**Testing instructions**:

1. Use `make test` in the OCIM virtual machine.

> CircleCI build status should be enough.

**Reviewers**
